operator-metadata/gating.yaml

```
@@ -1,9 +1,12 @@
+# https://gating-greenwave.readthedocs.io/en/latest/policies.html
 --- !Policy
-id: kafka-jenkins
+id: amq-streams-release
 product_versions:
-  - cvp
-decision_context: cvp_default
+  - amqstreams*
+decision_contexts:
+  - cvp_default
```

Bundles are supposed to have:
```suggestion
  - cvp_redhat_operator_default
```
but the change would bring stricter gating rules, so it may be best to leave it as is to avoid trouble